packages/sf/install

```diff
@@ -2,8 +2,13 @@
 set -x
 set -e
 
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0x089EBE08314DF160
-echo "deb http://ppa.launchpad.net/ubuntugis/ppa/ubuntu $(lsb_release -cs) main" >> /etc/apt/sources.list.d/ubuntugis-ppa.list
+OS_DISTRIBUTION=$(lsb_release -cs)
+
+# these sysdeps are available by default in bionic
```

Comment:
There is a desire to get the newer versions from the PPA which is why it isn't fetched from the main apt repo.